roles/sno_node_prep/tasks/15_validation_disconnected_registry.yml

```diff
@@ -7,14 +7,6 @@
   tags:
     - use_registry
 
-- name: Set fact disconnected_registry_mirrors_file
```

Comment:
This breaks anyone defining in the inventory the `disconnected_registry_mirrors_file` under registry_host, doesn't it?

Also breaks if someone is not explicitly defining `disconnected_registry_mirrors_file` when calling this role it will break when doing a disconnected environment, right?